composer.json

```diff
@@ -35,7 +35,7 @@
         "matomo/cache": "~2.0",
         "matomo/decompress": "~2.0",
         "matomo/ini": "~2.0",
-        "matomo/matomo-php-tracker": "dev-4.x-dev",
+        "matomo/matomo-php-tracker": "dev-ecommerceview",
```

Comment:
This needs to be reverted after https://github.com/matomo-org/matomo-php-tracker/pull/68 has been merged